src/screens/datasharing/components/BaseTekUploadView.tsx

```diff
@@ -8,7 +8,8 @@ import {covidshield} from 'services/BackendService/covidshield';
 import {xhrError} from 'shared/fetch';
 import AsyncStorage from '@react-native-community/async-storage';
 import {INITIAL_TEK_UPLOAD_COMPLETE, ContagiousDateInfo, ContagiousDateType} from 'shared/DataSharing';
-import {EventTypeMetric, FilteredMetricsService} from 'services/MetricsService/FilteredMetricsService';
+import {FilteredMetricsService} from 'services/MetricsService/FilteredMetricsService';
```

Comment:
Thinking we should setup an index file in services/MetricsService to pull these `FilteredMetricsService` and `EventTypeMetric` from to make this a single line import.

```javascript
import {FilteredMetricsService,EventTypeMetric} from 'services/MetricsService';
```

Reply:
You are right. I will change it.